programs/nodeos/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ target_link_libraries( ${NODE_EXECUTABLE_NAME}
         PRIVATE -Wl,${whole_archive_flag} txn_test_gen_plugin        -Wl,${no_whole_archive_flag}
         PRIVATE -Wl,${whole_archive_flag} db_size_api_plugin         -Wl,${no_whole_archive_flag}
         PRIVATE -Wl,${whole_archive_flag} producer_api_plugin        -Wl,${no_whole_archive_flag}
+        PRIVATE -Wl,${whole_archive_flag} resource_monitor_plugin    -Wl,${no_whole_archive_flag}
```

Comment:
You don't need `whole_archive_flag` for `resource_monitor_plugin` since you include and use it in main.

Reply:
I don't understand whole_archive_flag here. Should I change it to no_whole_archive_flag?

Reply:
His won't need the flag, but it's probably good to keep the flag for that plugin in case we change how and what directories it grabs.  If someone removes it from main, then they will spend probably way too much time trying figure out why it no longer works.  It can potentially make the binary slightly bigger, but I wouldn't mark that as a concern of ours.